packages/account/src/Sections/Security/LoginHistory/login-history-content.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Localize } from '@deriv/translations';
+import { Table } from '@deriv/components';
+import { TData } from '@deriv/api/src/hooks/useLoginHistory';
+import { fields } from '../../../../../utils/src/getLoginHistoryFormattedData';
+import LoginHistoryTableRow from './login-history-table-row';
+import LoginHistoryListRow from './login-history-list-row';
+
+type TGetFields = {
+    [key: string]: JSX.Element;
+};
+
+type TLoginHistoryContent = {
+    data: TData;
+    is_mobile: boolean;
+};
+
+const getFields = () =>
+    Object.fromEntries(
+        Object.entries(fields).map(([key, value]) => [
+            key,
+            <Localize i18n_default_text='{{value}}' values={{ value }} key={key} />,
```

Comment:
a doubt, will it work without passing values prop?
`<Localize i18n_default_text='{{value}}' key={key} />,`

Reply:
and will it work if we pass value to `i18n_default_text` directly, like
`... i18n_default_text={value} ....`

Reply:
@shaheer-deriv 
`<Localize i18n_default_text='{{value}}' key={key} />` in this scenario, the `i18n_default_text` has a string, which will not work without `values` prop as this component is a wrapper over `Trans` which only does interpolation of string